recipe/meta.yaml

```
@@ -1,59 +1,60 @@
-{% set name = "locust" %}
-{% set version = "0.8" %}
-{% set sha256 = "d0f2cf5c979d1c72f5f93cc5d7626e2fbe0c11031b22227370f7c7f7eeb4f5f3" %}
+{% set name = "locustio" %}
+{% set version = "0.11.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: locust
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 93404f831114791b0756325c53b08bff73f048eeb69688be657629feaa62b507
 
 build:
   noarch: python
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed -vv .
```

I think you can simply say:

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  
  see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33)
```